src/quequations.jl

```diff
@@ -14,23 +14,39 @@ end
 
 QuLiouvillevonNeumannEq{H<:QuBase.AbstractQuMatrix}(liouvillian::H) = QuLiouvillevonNeumannEq{H}(liouvillian)
 
-function liouvillian_op(h::QuBase.AbstractQuMatrix)
-    nb = size(coeffs(h), 1)
+immutable QuLindbladMasterEq{H<:QuBase.AbstractQuMatrix, V<:QuBase.AbstractQuMatrix} <: QuEquation
+    hamiltonian::H
+    collapse_ops::Vector{V}
+    QuLindbladMasterEq(hamiltonian, collapse_ops) = new(hamiltonian, collapse_ops)
+end
+
+QuLindbladMasterEq{H<:QuBase.AbstractQuMatrix, V<:QuBase.AbstractQuMatrix}(hamiltonian::H, collapse_ops::Vector{V}) = QuLindbladMasterEq{H,V}(hamiltonian, collapse_ops)
+
+function lindblad_op(hamiltonian::QuBase.AbstractQuMatrix, collapse_ops::Vector)
+    nb = size(coeffs(hamiltonian), 1)
+    nlop = length(collapse_ops)
+    heff = zeros(typeof(im*one(eltype(hamiltonian))), size(coeffs(hamiltonian)))
+    for l=1:nlop
+        heff = heff + 0.5*coeffs(collapse_ops[l])'*coeffs(collapse_ops[l])
+    end
     SI = Array(Int,0)
     SJ = Array(Int,0)
-    Lvals = Array(typeof(im*one(eltype(h))),0)
+    Lvals = Array(typeof(im*one(eltype(hamiltonian))),0)
     for m=1:nb
         for n=1:nb
             for i=1:nb
                 for j=1:nb
                     sm = (n-1)*nb + m
                     sj = (j-1)*nb + i
-                    lv = zero(Complex128)
+                    lv = zero(eltype(Lvals))
+                    for l=1:nlop
+                        lv = lv + collapse_ops[l][m,i]*collapse_ops[l]'[n,j]
```

Comment:
It's either `collapse_ops[l]'[j,n]` or `conj(collapse_ops[l][n,j])` as you can see for instance from the implementation in QuDOS.